### DISCUSSION TITLE
why I cannot download a song particular version if I can chose it to stream?

### DISCUSSION THREAD

**Discussion by ghost** [KRTirtho/spotube#534] [Q&A]
### So This the version v2.7.1 Build number 18
I was flabbergasted by this app and the features it upholds and I appreciate it very much so.
My fav feature was I can select alternative tracks for a particular song but the thing that makes it disappointing is that I am not able to Download that particular alternative version cause after selecting a version (Say **A**) from another version (Say **B**) and pressing download the track that downloads is still **B** 

_I just want this feature, or is it just I who is having this problem?_

**Comment by KRTirtho** [comment]
It's already implemented and available on nightly
